lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -13,10 +13,12 @@
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
+                tagName: 'create-new-team',
```

Comment:
This was being introduced in create team page, i've just moved it to here and looks like this is not being used anywhere. Removing it.